src/tests/JIT/Methodical/fp/exgen/10w5d_cs_do.csproj

```diff
@@ -3,6 +3,7 @@
     <!-- Needed for JIT/superpmi/superpmicollect.csproj which depends on the
          individual test .cmd file -->
     <RequiresProcessIsolation>true</RequiresProcessIsolation>
+    <IlasmRoundTripIncompatible>true</IlasmRoundTripIncompatible>
```

Comment:
Convention is to include reason(s) in the RequiresProcessIsolation comment, for which IlasmRoundTripIncompatible is a new one.

Are these due to ilasm/ildasm bugs that should have issues?  Or something else?